src/components/ebay-filter-menu/template.marko

```diff
@@ -0,0 +1,55 @@
+<script marko-init>
+    var processHtmlAttributes = require("../../common/html-attributes");
+</script>
+
+<var baseClass = data.withButton ? 'filter-menu-button': 'filter-menu'/>
```

Comment:
I'm confused about why the `filter-menu` is aware of the `filter-menu-button`.  Shouldn't it be completely isolated, and just pass events back to the button?

Reply:
Well ... it has to do with class names, and the way Skin was done. Let's discuss.